src/Pact/Analyze/Util.hs

```diff
@@ -0,0 +1,62 @@
+{-# language FlexibleContexts #-}
+
+module Pact.Analyze.Util where
+
+import qualified Data.Default              as Default
+import           Pact.Types.Lang           (Info(_iInfo), Parsed)
+import           Pact.Types.Typecheck      (AST(_aNode), Node(_aId), _tiInfo)
+
+(<$$>) :: (Functor f, Functor g) => (a -> b) -> f (g a) -> f (g b)
+(<$$>) = fmap . fmap
+
+(<$$$>)
+  :: (Functor f, Functor g, Functor h)
+  => (a -> b) -> f (g (h a)) -> f (g (h b))
+(<$$$>) = fmap . fmap . fmap
+
+(<$$$$>)
+  :: (Functor f, Functor g, Functor h, Functor i)
+  => (a -> b) -> f (g (h (i a))) -> f (g (h (i b)))
+(<$$$$>) = fmap . fmap . fmap . fmap
```

Comment:
caaaadr anyone? Not sure about the shark jumping here but not super-opinionated about it

Reply:
Updated with less shark jumping